solidity/random-beacon/package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "clean": "hardhat clean",
     "build": "hardhat compile",
-    "test": "hardhat test",
+    "check-accounts-count": "hardhat check-accounts-count",
```

Comment:
We don't need to define this script. It's enough that we included task execution in `test` script.
```suggestion
```

Reply:
right, removed in [fcbfa94](https://github.com/keep-network/keep-core/pull/2958/commits/fcbfa946e9f1c860de440c12e5eeb05a24965322)